main.go

```diff
@@ -172,6 +173,16 @@ func _main(ctx context.Context) error {
 			DBName:       cfg.PGDBName,
 			QueryTimeout: cfg.PGQueryTimeout,
 		}
+		// Replace {netname} with curnet
+		if strings.Contains(dbi.DBName, "{netname}") {
+		  reg, err := regexp.Compile("[^a-zA-Z]+")
+		  if err != nil {
+		    return err
+		  }
+		  
+		  var networkName = reg.ReplaceAllString(curnet.String(), "");
+		  dbi.DBName = strings.Replace(dbi.DBName, "{netname}", networkName, 1);
```

Comment:
Best to define a single constant for `"{netname}"` since it is written as a literal twice.